src/server/knexfile.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
       user: process.env.DB_USER,
       password: process.env.DB_PASSWORD,
       database: process.env.DB_NAME,
+      dateStrings: true,
```

Comment:
What does this do? Force the DB to return strings? Why is that necessary? I would think it would be better to keep dates as timestamps for as long as possible and only render them into strings in the presentational components.